<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCRdataDialog</class>
 <widget class="QDialog" name="TCRdataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="query_tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>363</width>
     <height>721</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>TCR</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="Time_radioButton">
           <property name="text">
            <string>时间查询</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="St_pushButton">
               <property name="text">
                <string>开始时间</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Et_pushButton">
               <property name="text">
                <string>结束时间</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QDateTimeEdit" name="St_dateTimeEdit"/>
             </item>
             <item>
              <widget class="QDateTimeEdit" name="Et_dateTimeEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="Name_radioButton">
           <property name="text">
            <string>名称查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="Query_pushButton">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Display_pushButton">
           <property name="text">
            <string>打印</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_pushButton">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pdf_pushButton">
           <property name="text">
            <string>PDF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>外部测量</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QRadioButton" name="PTime_radioButton">
           <property name="text">
            <string>时间查询</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="PSt_pushButton">
               <property name="text">
                <string>开始时间</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="PEt_pushButton">
               <property name="text">
                <string>结束时间</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QDateTimeEdit" name="PSt_dateTimeEdit"/>
             </item>
             <item>
              <widget class="QDateTimeEdit" name="PEt_dateTimeEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QRadioButton" name="PName_radioButton">
           <property name="text">
            <string>名称查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="p_lineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="PQuery_pushButton">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PDisplay_pushButton">
           <property name="text">
            <string>打印</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pdelete_pushButton">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Ppdf_pushButton">
           <property name="text">
            <string>PDF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="p_tableWidget"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
